src/components/content/content2.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+<div class="container">
+  <div class="item item_image">
+    <img :src="setting[0].photo">
+  </div>
+  <div class="item item_content">
+    <div class="item texto">
+      <h1>{{  setting[0].title }}</h1>
+      <p>{{ setting[0].description }}</p>
+      <a :href="setting[0].redirect_url">
+        <button class="button">{{setting[0].button_text}}</button>
+      </a>
+    </div>
+  </div>
+  <div class="item item_content item_last">
```

Comment:
En el caso que tiene un arreglo podria utilizar un v-for para no repetir dos o mas veces el html